src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1449,6 +1449,90 @@ void valid() {
                 assertAndDump(model, "NIST.valid: confidence must be between 0 and 1", nistSeedlingValidator, true);
             }
         }
+
+        // CompoundJustification must be used only for justifications of argument assertions,
+        // and not for justifications for entities, events, or relation KEs
+        @Nested
+        class RestrictCompoundJustification {
+            @Test
+            void invalid() {
```

Comment:
This test fails for more reasons than just the justification. Please remove all other failures (non-Entity type and Entity/Event/Relation not clustered). You probably don't even need to create a new Entity/Event/Relation for this. You could just use the class members.

Reply:
Updated to use the class members. Currently there is no relation class member object so I used the one created in the merged code from the EdgeJustificationCompound invalid test. Let me know if you would prefer I create a new class member for relation. I know this done in a future PR.